test/e2e/helpers.js

```diff
@@ -57,7 +57,7 @@ async function withFixtures(options, testSuite) {
   const bundlerServer = new Bundler();
   const https = await mockttp.generateCACertificate();
   const mockServer = mockttp.getLocal({ https, cors: true });
-  let secondaryGanacheServer;
+  const secondaryGanacheServer = [];
```

Comment:
wonder if can we combine `ganacheServer` and `secondaryGanacheServer` into a single `ganacheServers`